agent/bench-scripts/pbench-uperf

```diff
@@ -402,36 +426,39 @@ if [ ! -z "$clients" -a "$postprocess_only" != "y" ]; then
 		error_log "    vsock-servers(${v_cnt}): ${vsock_servers}"
 		exit 1
 	fi
-fi
-# before we run a test, verify clients and servers are accessible and install
-# uperf if necessary
-if [ "$postprocess_only" != "y" ]; then
-	# Always install locally
-	# FIXME: why is this necessary?
-	check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+
+	if [[ ${check_local} -gt 0 ]]; then
+		check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+	fi
+
 	let err_cnt=0
 	err_clients=""
+	_clients=${clients:-localhost}
```

Comment:
1. Isn't this redundant since `clients` is already set to `clients_def` (`127.0.0.1`) if it wasn't specified?
2. You don't actually seem to be using `_clients` anyway. Did I miss it?

Reply:
It appears to be unused.  Removed.